gcp-pubsub/src/test/groovy/io/micronaut/gcp/pubsub/bind/ReactiveConsumerSpec.groovy

```diff
@@ -212,13 +212,13 @@ class ReactiveConsumer {
     @Subscription("mono-result")
     Mono<String> onMessage1(PubsubMessage message) {
         this.msg = message
-        return Mono.just(message).delayElement(Duration.ofSeconds(2)).then(Mono.just("success")).doOnTerminate { finished.set(true) }
+        return Mono.just(message).delayElement(Duration.ofMillis(200)).then(Mono.just("success")).doOnTerminate { finished.set(true) }
```

Comment:
why are we changing these tests in this PRs from 2 seconds to milliseconds? To make these tests faster?